Directory.Build.props

```diff
@@ -12,7 +12,7 @@
   <Import Project="eng\FacadeAssemblies.props" />
   <Import Project="eng\Versioning.props"/>
   <Import Project="eng\TestProperties.props"/>
-  <Import Project="eng\uwp.props" />
+  <Import Project="eng\targetFramework.props" />
```

Comment:
>t [](start = 23, length = 1)

Nit: capitalize file names to make them consistent.

Reply:
Will do.